spec/light-client/verification/verification_001_published.md

```diff
@@ -734,7 +744,11 @@ func VerifyToTarget(primary PeerID, lightStore LightStore,
         }
 
         // Verify
-        verdict = ValidAndVerified(lightStore.LatestVerified, current)
+        if sequential {
```

Comment:
What is `sequential` here?

Reply:
a boolean config variable that is part of the light client. I've just glossed over that part

Reply:
I see. There is a section `### Variables`. Could you please add `sequential` there and explain the initialization/assumption there.

Reply:
Done